compiler/src/iree/compiler/Codegen/Dialect/Codegen/IR/IREECodegenDialect.cpp

```diff
@@ -45,4 +46,39 @@ void IREECodegenDialect::initialize() {
       >();
 }
 
+LogicalResult
+IREECodegenDialect::verifyOperationAttribute(Operation *op,
+                                             NamedAttribute attribute) {
+  StringRef symbol = attribute.getName().strref();
+  Attribute attr = attribute.getValue();
+
+  // Early return if the symbol is not "iree_codegen.tuning_spec_entrypoint"
```

Comment:
I don't think this comment adds clarity, I'd drop it. Instead, you can summarize the validity criteria in one longer comment.